druid/ingestion-spec/rollups/ml_project_index_rollup.json

```diff
@@ -0,0 +1,350 @@
+                "type": "path",
+                "name": "tasks_status",
+                "expr": "$.tasks[*].status"
+              },
+              {
+                "type": "path",
+                "name": "designation",
+                "expr": "$.userRoleInformation.role"
+              },
+              {
+                "type": "path",
+                "name": "task_evidence",
+                "expr": "$.tasks[*].attachments[*].sourcePath"
+              },
+              {
+                "type": "path",
+                "name": "task_id",
+                "expr": "$.tasks[*]._id"
+              },
+              {
+                "type": "path",
+                "name": "program_id",
+                "expr": "$.programId"
+              },
+              {
+                "type": "path",
+                "name": "status_of_project",
+                "expr": "$.status"
+              },
+              {
+                "type": "path",
+                "name": "program_name",
+                "expr": "$.programInformation.name"
+              },
+              {
+                "type": "path",
+                "name": "project_updated_date",
+                "expr": "$.updatedAt"
+              },
+              {
+                "type": "path",
+                "name": "createdBy",
+                "expr": "$.userid"
+              },
+              {
+                "type": "path",
+                "name": "program_externalId",
+                "expr": "$.programExternalId"
+              },
+              {
+                "type": "path",
+                "name": "private_program",
+                "expr": "$.isAPrivateProgram"
+              },
+              {
+                "type": "path",
+                "name": "project_terms_and_condition",
+                "expr": "$.hasAcceptedTAndC"
+              },
+              {
+                "type": "path",
+                "name": "solution_id",
+                "expr": "$.solutionInformation._id"
+              },
+              {
+                "type": "path",
+                "name": "state_externalId",
+                "expr": "$.userRoleInformation.state"
+              },
+              {
+                "type": "path",
+                "name": "block_externalId",
+                "expr": "$.userRoleInformation.block"
+              },
+              {
+                "type": "path",
+                "name": "district_externalId",
+                "expr": "$.userRoleInformation.district"
+              },
+              {
+                "type": "path",
+                "name": "cluster_externalId",
+                "expr": "$.userRoleInformation.cluster"
+              },
+              {
+                "type": "path",
+                "name": "school_externalId",
+                "expr": "$.userRoleInformation.school"
+              }
+            ]
+          },
+          "featureSpec": {}
+        },
+        "replicas": 1,
+        "taskCount": 1,
```

Comment:
The taskCount needs to be injected using a variable so that we can change the taskCount if we want to scale the ingestion.

Reply:
This is fixed now. Please check